[
    {
        "content": "<p>Hi! I've been looking into <a href=\"https://github.com/rust-lang/rust/issues/50507\">https://github.com/rust-lang/rust/issues/50507</a>, linked from the Query Parallelization Tracking Issue.<br>\nI thought I'd start by trying to reproduce the mentioned crash. I've replaced <code>.insert</code> with <code>.insert_same</code>. Then I ran the whole compiler test suite and all tests passed... so I'm guessing this bug doesn't exist anymore?<br>\nIs there anything I'm doing wrong?</p>",
        "id": 202118103,
        "sender_full_name": "Gabriel Majeri",
        "timestamp": 1593192069
    },
    {
        "content": "<p>I don't know that it makes too much sense to look at fixing that issue, presuming you're in parallel mode, because parallel compiler is not an area of active effort right now</p>",
        "id": 202129507,
        "sender_full_name": "simulacrum",
        "timestamp": 1593198402
    },
    {
        "content": "<p>but if you can't reproduce it with parallel enabled we can perhaps just close</p>",
        "id": 202129547,
        "sender_full_name": "simulacrum",
        "timestamp": 1593198430
    }
]